drivers/bridge/port_mapping_test.go

```diff
@@ -97,6 +97,8 @@ func TestPortMappingConfig(t *testing.T) {
 }
 
 func TestPortMappingV6Config(t *testing.T) {
+	t.Skip("FIXME: circleci does not have proper IPv6 support")
```

Comment:
I'll open a PR to revert this skip and do further digging into what's missing on these nodes;

The CircleCI environment may not have a usable IPv6, causing the test
to fail:

    === RUN   TestHostsEntries
    ERRO[0000] Could not add route to IPv6 network fe90::1/64 via device test_nw_0: network is down 
    
    === RUN   TestPortMappingV6Config
    WARN[0001] bridge store not initialized. kv object docker/network/v1.0/bridge/dummy/ is not added to the store 
    WARN[0001] bridge store not initialized. kv object docker/network/v1.0/bridge-endpoint/ep1/ is not added to the store 
    WARN[0001] bridge store not initialized. kv object docker/network/v1.0/bridge-endpoint/ep1/ is not added to the store 
        port_mapping_test.go:155: Failed to store the port bindings into the sandbox info. Found: [{udp 172.19.0.11 400 0.0.0.0 54000 54000} {tcp 172.19.0.11 500 0.0.0.0 65000 65000} {sctp 172.19.0.11 500 0.0.0.0 65000 65000}]
    --- FAIL: TestPortMappingV6Config (0.12s)


So either we need to change something in the config (or test setup), or we need to properly detect if  it's supported (so can be tested, or should be skipped)